go/governance/gen_vectors/main.go

```diff
@@ -40,14 +38,39 @@ func main() {
 			// Valid submit upgrade proposal transaction.
 			for _, epoch := range []uint64{0, 1000, 10_000_000} {
 				for _, name := range []string{"", "descriptor-name"} {
-					for _, identifier := range []string{hex.EncodeToString([]byte{}), hex.EncodeToString(emptyHash[:]), "000000000000000000000000000000000000000000000000000000000abcdefg"} {
+					for _, identifier := range []version.ProtocolVersions{
```

Comment:
cc @tjanez when this PR is finalized/merged we should communicate this change for the submit proposal tx format to Juan